manager.py

```diff
@@ -373,7 +429,15 @@ def from_argparse(cls):
             '--port', '-p', type=int, default=DEFAULT_PORT,
             help='Specify a port for the server (default {})'.format(
                 DEFAULT_PORT))
-
+        #datalog
+        parser.add_argument(
+            '--datalog', '-d', default=None,
+            help='Specify a path for the datalog (default {})'.format(
+                None))
```

Comment:
Actually put `DEFAULT_DATALOG` in this `help` line instead of `None`. Even though it's technically `None`, it's more helpful to the user to see what it would be set as if you do `-a`